subsys/bluetooth/mesh/access.c

```diff
@@ -465,15 +466,71 @@ struct bt_mesh_elem *bt_mesh_elem_find(uint16_t addr)
 		}
 	}
 
+	return NULL;
+}
```

Comment:
Suggest restructuring this with early exits:
```c
if (!BT_MESH_ADDR_IS_UNICAST(addr) {
    return NULL;
}

index = addr - dev_comp->elem[0].addr;
if (index >= dev_comp->elem_count) {
    return NULL;
}

return &dev_comp->elem[index];
```

Reply:
Yes, I agree. I'll change the structure.